riff-raff/public/docs/riffraff/s3buckets.md

```diff
@@ -1,67 +0,0 @@
-<!--- prev:externalRequest -->
-Riff-Raff, AWS credentials and S3 uploads
```

Comment:
I think this is now covered by https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/riffraff/riff-raff-roles.md